tests/SpecTests/ClientSideEncryptionSpecTest.php

```diff
@@ -167,7 +165,7 @@ public function testClientSideEncryption(stdClass $test, ?array $runOn, array $d
             $this->markTestIncomplete(self::$incompleteTests[$this->dataDescription()]);
         }
 
-        if (str_starts_with($this->dataDescription(), 'fle2v2-Range-') && version_compare(phpversion('mongodb'), '1.20.0dev', '>=')) {
+        if (str_starts_with($this->dataDescription(), 'fle2v2-Range-')) {
```

Comment:
This test files could be removed then...

Reply:
Agreed - these are no longer necessary. They are also no longer present in the spec tests, but we probably forgot to remove them.

Reply:
Already removed by https://github.com/mongodb/mongo-php-library/pull/1350